app/models/sitcom_manager.rb

```diff
@@ -0,0 +1,42 @@
+class SitcomManager
+  attr_reader :search_params, :page_params
+
+  def initialize(search_params, page)
```

Comment:
Unused method argument - `page`. If it's necessary, use `_` or `_page` as an argument name to indicate that it won't be used.